query/accident_queries_optimize.py

```diff
@@ -0,0 +1,143 @@
+from tabulate import tabulate
+from rdflib import Graph
+import pandas as pd
+import time
+import traceback
+
+accident_rdf_location = "./data/rdf/"
+output_location = "./query/output/"
+accidentNamespace = 'http://github.com/kawai924/SementicNYWeatherAccident/accident#'
+stationsNamespace = 'http://github.com/kawai924/SementicNYWeatherAccident/station#'
+weatherTypeNamespace = 'http://github.com/kawai924/SementicNYWeatherAccident/weather#'
+
+"""
+Query : Since, it is taking time to load the RDFs , built this query to load the RDFs once, and ask for input , 
+we can paste the python query in terminal and it will execute successfully giving the result if there is 
+no error OR it will throw an error if something is wrong.In either cases, it will again ask user to input query to run.
+Developers : Aditi Tomar and Gayathri Venna
+"""
+
+
+"""
+How-to query graph in rdflib using sparql: https://www.oreilly.com/library/view/programming-the-semantic/9780596802141/ch04.html
+"""
+def start():
+    
+    accident_graph = createGraph()
+
+    while True:
+        try:
+            execute(accident_graph)
+        except:
+            traceback.print_exc()
+            print("Exception occured. Try again")
+
+
+""" Replaces namespace in resource with prefix for shorter display in output data """
+def replace_prefix(data):
+    if not data:
+        return
+    elif accidentNamespace in data:
+        return data.replace(accidentNamespace, "act:")
+    elif stationsNamespace in data:
+        return data.replace(stationsNamespace, "sta:")
+    elif weatherTypeNamespace in data:
+        return data.replace(weatherTypeNamespace, "wea:")
+    else:
+        return data
+
+def createGraph():
```

Comment:
you should use the knowledge_graph singleton to get/create the graph. Once one query was executed, all other queries will just get the graph without reload all